packages/roosterjs-editor-api/lib/format/createLink.ts

```diff
@@ -83,6 +83,13 @@ export default function createLink(
             } else {
                 // the selection is not collapsed, use browser execCommand
                 editor.getDocument().execCommand(DocumentCommand.CreateLink, false, normalizedUrl);
+                const traverser = editor.getSelectionTraverser();
+                let currentInline = traverser.getNextInlineElement();
+                while (currentInline) {
+                    const a = currentInline.getContainerNode();
+                    a.parentElement.removeChild(a);

```

Comment:
So this looks like a bug from execCommand, it creates extra node, is that right?

Then this code can only work if this bug exists. If somehow browser fixed the bug, (or try other browsers like Firefox, Safari, Opera, see if they have the same issue), then this code may become another bug?

Reply:
I see, debugging this.

Reply:
I tried in multiple browsers and the bug that is causing execCommand to create multiple anchor elements instead of 1 is in Edge, Chrome, Opera and Firefox.

With the fix in this branch it is fixing the issue in all the above browsers.

If there is a fix for this bug in execCommand, I think this code should not create another issue since we only remove the elements that are in selection and that are after the first anchor element. 

What do you think?